tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+          "explanation": "Designed for students who wish to focus on art, film, music, sound technology, and/or media theory broadly conceived. Students take a minimum of five courses in the German Department (usually no more than two at the 200 level), and a maximum of three relevant cognate courses in Art & Archaeology, Music, Philosophy, European Cultural Studies, and the Program in Visual Arts.",
+          "req_list": [
+            {
+              "name": "German",
+              "max_counted": null,
+              "min_needed": 5,
+              "explanation": "Students take a minimum of five courses in the German Department (usually no more than two at the 200 level).",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 3*",
+                    "GER 4*",
+                    "GER 5*"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 3,
+              "min_needed": null,
+              "explanation": "A maximum of three relevant cognate courses in Art & Archaeology, Music, Philosophy, European Cultural Studies, and the Program in Visual Arts.",
+              "course_list": [
+                "ART *",
+                "MUS *",
+                "PHI *",
+                "ECS *",
+                "VIS *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Linguistics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
+          "req_list": [
+            {
+              "name": "Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
+              "course_list": [
+                "LIN 213",

```

This seems like a typo on the department's end, right?
"Introduction to Language and Linguistics" is actually LIN 201, while LIN 213 is "Languages and Culture in South Asia" which seems quite irrelevant.
```suggestion
                "LIN 201",
```

---

Oh, it seems the [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf) got it right and says LIN 201, so this is almost definitely a typo.